packages/devtools_app/lib/src/debugger/flutter/debugger_screen.dart

```diff
@@ -210,12 +216,19 @@ class DebuggerScreenBodyState extends State<DebuggerScreenBody>
           ),
           color: titleSolidBackgroundColor,
         ),
-        padding: const EdgeInsets.only(left: defaultSpacing),
+        padding: const EdgeInsets.only(left: defaultSpacing, right: 4.0),
         alignment: Alignment.centerLeft,
         height: DebuggerScreen.debuggerPaneHeaderHeight,
-        child: Text(
-          title,
-          style: Theme.of(context).textTheme.subtitle2,
+        child: Row(
+          children: <Widget>[
```

Comment:
remove `<Widget>`

Reply:
My code completion is very insistent here for some reason.

Reply:
Ah!

@pq, I still see the generic types inserted with the 'wrap with' refactorings (Row, Column, ...).

Reply:
Ah!  Missed that.

Looking now (https://github.com/dart-lang/sdk/issues/41517).